src/modules/processes.nix

```diff
@@ -4,7 +4,7 @@ let
   processType = types.submodule ({ config, ... }: {
     options = {
       exec = lib.mkOption {
-        type = types.str;
+        type = types.lines;
```

Comment:
I'm not sure we should change it to lines, that means multiple definitions will get merged, resulting into weird errors.

Can't we just keep `str`?

Reply:
sure thing!